client/app/index.html

```diff
@@ -221,6 +222,8 @@ <h3 translate>About</h3>
             'liveblog.posts',
             'liveblog.blog',
             'liveblog.themes',
+            'liveblog.freetypes',
```

Comment:
I agree that using `liveblog.freetypes` and `liveblog.freetype` as module names can be confusing